triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -75,6 +76,8 @@ object ServiceMain {
           allowExistingSchema,
```

Comment:
Shall we name the above method as `startServerForTest` and make it package private?